doc/rados/configuration/bluestore-config-ref.rst

```diff
@@ -294,3 +294,29 @@ set with::
 :Type: Unsigned Integer
 :Required: No
 :Default: 64K
+
+SPDK Usage
+==================
+
+If you want to use SPDK driver for NVME SSD, you need to specify NVMe serial
+number here with "spdk:" prefix for ``bluestore_block_path``.
+Users can use 'lspci -vvv -d 8086:0953 | grep "Device Serial Number"' to
+get the serial number of Intel(R) Fultondale NVMe controllers.
+Example:
+bluestore_block_path = spdk:55cd2e404bd73932
+If you want to run multiple SPDK instances per node, you must specify the
+amount of dpdk memory size in MB each instance will use, to make sure each
+instance uses its own dpdk memory
+
+In most cases, we only need one device to serve as data, db, db wal purposes.
+We need to set configurations below to make sure all IOs issued under SPDK.::
```

Comment:
I think thi si sonly needed for vstart, right? noramlly these default to empty, but vstart.sh sets them up (mostly just to exercise the code; not for a good reason)

Reply:
bluestore_block_wal_size default is not zero. let user make sure these configurations is ok.

Reply:
but the _create options should be false, and those are only used by vstart for dev purposes.  since spdk shouldn't be used with multiple devices it's probably simpler to leave that out to avoid confusion?

Reply:
if we're worried about a vstart user having problems with spdk i'd rather disable wal/db for vstart to avoid it, and/or add an --spdk <devid> option to help

Reply:
hmm, let's remove _create options